### PR TITLE
add new device ACS-MSN4700

### PR DESCRIPTION
**Pull request by keboliu**
**- What I did**

Add new device ACS-MSN4700:
sai.profile, port_config.ini, buffer configuration.

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: device/mellanox/x86_64-mlnx_msn4700-r0/default_sku [hunk 1]
```diff
@@ -0,0 +1 @@
+ACS-MSN3700 t1
```
Comment (andriymoroz-mlnx) [inline device/mellanox/x86_64-mlnx_msn4700-r0/default_sku]: 37->47

File: device/mellanox/x86_64-mlnx_msn4700-r0/ACS-MSN4700/qos.json.j2 [hunk 2]
```diff
@@ -0,0 +1 @@
+../../x86_64-mlnx_msn2700-r0/ACS-MSN2700/qos.json.j2
```
Comment (andriymoroz-mlnx) [inline device/mellanox/x86_64-mlnx_msn4700-r0/ACS-MSN4700/qos.json.j2]: 27->47

Comment (andriymoroz-mlnx) [reply device/mellanox/x86_64-mlnx_msn4700-r0/ACS-MSN4700/qos.json.j2]: or it the same?

File: device/mellanox/x86_64-mlnx_msn4700-r0/default_sku (hunk 1 above)
Comment (keboliu) [reply device/mellanox/x86_64-mlnx_msn4700-r0/default_sku]: fixed.

File: device/mellanox/x86_64-mlnx_msn4700-r0/ACS-MSN4700/qos.json.j2 (hunk 2 above)
Comment (keboliu) [reply device/mellanox/x86_64-mlnx_msn4700-r0/ACS-MSN4700/qos.json.j2]: they are the same.

File: device/mellanox/x86_64-mlnx_msn4700-r0/sensors.conf [hunk 3]
```diff
@@ -0,0 +1,116 @@
+################################################################################
+# Copyright (c) 2019 Mellanox Technologies
+#
+# Platform specific sensors config for SN3700
```
Comment (andriymoroz-mlnx) [inline device/mellanox/x86_64-mlnx_msn4700-r0/sensors.conf]: another 3700
also sensors list can be different. We probably don't know yet...

**Pull request closed without merging**